hack/custom-resources/authgear.secrets.yaml

```diff
@@ -3,10 +3,6 @@ secrets:
     data:
       database_schema: app
       database_url: 'postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable'
-  - data:
```

Comment:
Actually I never use this file. How are you using it?

Reply:
This file set secrets for all projects. Removing the audit db secret here so that new projects will not have audit db secret.